main/define/src/mill/define/Cross.scala

```diff
@@ -128,6 +128,7 @@ object Cross {
   )
 
   object Factory {
+    import scala.language.implicitConversions
```

Comment:
Why do we need implicit conversions?

Reply:
This is to silence a compiler warning already present before this change.